tests/integration/test_vulnerability_detector/test_cpe_helper/test_cpe_helper_missing_fields.py

```diff
@@ -0,0 +1,256 @@
+'''
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Wazuh is able to detect vulnerabilities in the applications installed in agents using the Vulnerability Detector
```

Comment:
Same description than `test_cpe_helper_fields`. Why are in different modules?

Reply:
Done [here](https://github.com/wazuh/wazuh-qa/pull/3731/commits/e6b49a5a140e4b5100cf848d49091abf9eb31df6)